src/_imagingcms.c

```diff
@@ -26,15 +26,11 @@ kevin@cazabon.com\n\
 https://www.cazabon.com\n\
 "
 
-#define PY_SSIZE_T_CLEAN
-#include "Python.h"  // Include before wchar.h so _GNU_SOURCE is set
-#include "wchar.h"
-#include "datetime.h"
+#include "libImaging/Imaging.h"  // Include before wchar.h so _GNU_SOURCE is set
```

Comment:
```suggestion
#include "libImaging/Imaging.h"  // Set _GNU_SOURCE in Python.h before wchar.h
```
To communicate that Python.h is the relevant part of Imaging.h